app/views/content/partials/teaching/_funding.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="funding">
+  <h3>Fund your teacher training</h3>
+  <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_chemistry %> or scholarships of <%= v :scholarships_chemistry %></a> are available for eligible trainee <%= @front_matter["subject"] %> teachers.</p>
+  <p>You can get a bursary or scholarship alongside a tuition fee and maintenance loan. Even if you've had financial support for a previous course, you could still be eligible for funding to support your teacher training.</p>
+  <p>You may also be able to get <a href="/funding-and-support">extra funding and support</a> if you're:</p>
+  <ul>
+    <li>a parent or carer</li>
+    <li>disabled</li>
+    <% if @front_matter["veteran"] %>
+      <li>a veteran</li>
+    <% end %>
```

Comment:
@sarahcrack Hiya, had a chat with Gemma about this and the veteran-specific info is aimed at undergrads, so given this is a page for post-graduates, we wouldn't need this part and veterans can just be included as a regular bullet point and appear on every page.
BUT it is a fabulous piece of code that I'm sure we'll be able to use elsewhere on the site for other partials!